sdk/core/ts-http-runtime/test/client/clientHelpers.spec.ts

```diff
@@ -27,8 +27,8 @@ describe("clientHelpers", () => {
     );
   });
 
-  it("should create a default pipeline with apiVersion policy", () => {
-    const pipeline = createDefaultPipeline(mockBaseUrl);
+  it("should create a pipeline with apiVersion policy when one is defined", () => {
```

Comment:
wonder if we should add a counter case that there's no apiVersion policy if no apiVersion is passed?